sources/models/src/aws-k8s-1.20

```diff
@@ -0,0 +1 @@
+aws-k8s-1.15
```

Comment:
```suggestion
aws-k8s-1.19
```
To match the existing pattern; I'm not opposed to changing the pattern, but while it's there we should be consistent.  (I think the pattern does help in case we determine we need to change the model for 1.18+, for example; we can just change 1.18 and not need to repoint the links following it.)

Reply:
Possibly with symlinks inside, and an override / customization file, rather than duplicating the rest.

Reply:
The files in `defaults.d` were already symlinked. I symlinked `mods.rs` to aws-k8s-1.19's `mods.rs`